### ISSUE TITLE
Sent with badge not always aligned

### ISSUE THREAD

**Issue by Huddie** [GitHawkApp/GitHawk#2010]
This is a question.
The sent with badge is clearly attached to timestamp so when the timestamp is a single digit vs a double digit the sent with icon will be in different places.

This makes it the sent with badges look unaligned when scrolling 
Not sure if this is on purpose. I do know this is **not** high priority if it wasn’t on purpose.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Good point. They should all be aligned. It’s pinned to the left of the time stamp, but it should be fixed width from the right of the view

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Meta example

![GitHawk Upload by rnystrom](https://i.imgur.com/le6zmJM.jpg)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
